documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -9,57 +13,92 @@
-Over time, the change events captured by {prodname} connectors and written by Kafka Connect into a topic may have different versions of the same schema,
-and Avro serialization makes it far easier for consumers to adapt to the changing schema.
+* In the Kafka Connect worker configuration. 
++
+In this case, the same converters are used for all connectors that are deployed to that worker's cluster.
 
-== The Apicurio API and Schema Registry
+* For an individual connector. 
 
-The open-source project https://github.com/Apicurio/apicurio-registry[Apicurio Registry] provides several components that work with Avro:
+Kafka Connect comes with a _JSON converter_ that serializes message keys and values into JSON documents. You can configure the JSON converter to include or exclude the message schema by specifying the `key.converter.schemas.enable` and `value.converter.schemas.enable` properties. The {prodname} {link-prefix}:{link-tutorial}[tutorial] shows what the messages look like when both payload and schemas are included. 
 
-* An Avro converter that can be used in Kafka Connect workers to map the Kafka Connect schemas into Avro schemas and to then use those Avro schemas to serialize the message keys and values into the very compact Avro binary form.
-* An API/Schema registry that tracks all of the Avro schemas used in Kafka topics, and where the Avro Converter sends the generated Avro schemas.
-Since the Avro schemas are stored in this registry, each message need only include a tiny _schema identifier_.
-This makes each message even smaller, and for an I/O bound system like Kafka this means more total throughput of the producers and consumers.
-* Avro _Serdes_ (serializers and deserializers) for Kafka producers and consumers.
-Any Kafka consumer applications you write to consume change events can use the Avro Serdes to deserialize the changes events.
+Including schemas causes the messages to  be very verbose. If you want your messages serialized with JSON, consider setting these properties to `false` to exclude the verbose schema information.
 
-You can install them into any Kafka distribution and use them with Kafka Connect.
+Alternatively, you can serialize the message keys and values by using link:https://avro.apache.org/[Apache Avro].
+The Avro binary format is compact and efficient, and Avro schemas make it possible to ensure that the messages have the correct structure. Avro's schema evolution mechanism makes it possible to evolve the schemas over time, which is essential for {prodname} connectors that dynamically generate the message schemas to match the structure of the database tables. Over time, the change events captured by {prodname} connectors and written by Kafka Connect into a topic may have different versions of the same schema. Avro serialization makes it easier for consumers to adapt to the changing schema.
```

Comment:
Without the section before (which now is part of the Apicurio section), there's some context lacking _why_ a registry is needed. How about adding something like this here:

When using Apache Avro as a message format, a schema registry must be deployed which manages Avro message schemas and their versions. Options include the Apicurio API and Schema Registry and the Confluent Schema registry.

Reply:
Before this point in the source file, no content is conditionalzed for just upstream nor just downstream. That is, all content is visible both upstream and downstream. But I think you are right that the doc needs to say that a registry is required and why. I added the following at the end of the section: 

To use Apache Avro as the message format, you must deploy a schema registry that manages Avro message schemas and their versions.

I left out you suggested second sentence because there is only one option downstream, and the next section in the source file describes it. Then a bit later, conditionalized for upstream only, the doc says you can use Confluent.

Reply:
Sounds good.